flottform/where-are-you-at/src/routes/+page.svelte

```diff
@@ -99,3 +125,35 @@
 		Error connecting to friend
 	{/if}
 </div>
+
+<style>
+	.copy-button-link-wrapper {
+		display: flex;
+		flex-direction: row-reverse;
```

Comment:
I kind of like it more when the clipboard button is on the left, but that's a matter of taste. If you prefer it on the right, that's fine too 😅 

```suggestion
		flex-direction: row;
```